pkgs/tools/package-management/nix/default.nix

```diff
@@ -273,4 +273,33 @@ in rec {
 
   });
 
+  nix_2_6 = lib.lowPrio (callPackage common rec {
+    pname = "nix";
+    version = "2.6.1";
+    src = fetchFromGitHub {
+      owner = "NixOS";
+      repo = "nix";
+      rev = "2.6.1";
```

Comment:
could be `rev = version;`. which it is for `nix_2_7`, so that would be consistent.

Reply:
I've just gone and removed nix_2_6 completely since it's not supported anyway :)

Reply:
Did you forget to push that commit?

Reply:
I did, oops...